src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)